beacon_node/network/src/metrics.rs

```diff
@@ -218,13 +218,30 @@ lazy_static! {
     /*
      * Bandwidth metrics
      */
+    pub static ref INBOUND_LIBP2P_QUIC_BYTES: Result<IntGauge> =
+        try_create_int_gauge("libp2p_inbound_quic_bytes", "The inbound quic bandwidth over libp2p");
```

Comment:
This metrics only ever goes up, right? Shouldn't it be a counter instead of a gauge?

Reply:
I think with counters we have to increase the counter, with gauges we can set the value. I think @jxs has to set the value based on the bandwidth at regular intervals. I'll let him comment tho. 

Agree that a counter would be better if we can set it easily.

Reply:
You may use [`Counter::inner`](https://docs.rs/prometheus-client/latest/prometheus_client/metrics/counter/struct.Counter.html#method.inner). Note that using a _Gauge_ in a place where a _Counter_ is correct might result in problems e.g. around Grafana's query suggestions.